### DISCUSSION TITLE
Can't add authenticator 2FA

### DISCUSSION THREAD

**Discussion by jeffbw13** [community/community#135200] [New to GitHub] [answered]
Currently I use mobile app for 2fa.  I would like to add TOTP in case I ever don't have my phone.  I have Authenticator extension on chrome.  I believe I need to add github to it as an account, but authenticator wants either a QR code or a Secret, and I don't know how to get either one.

**Comment by jeffbw13** [comment, marked as answer]
Never mind, I had to re-add my previously set-up authenticator info for some reason.  Was able to redo it by signing in using mobile.  Whew!

**Comment by Luckk56** [comment]
https://x.com/chrettonbase

**Comment by Luckk56** [comment]
https://x.com/chrettonbase

**Comment by Luckk56** [comment]
https://x.com/chrettonbase

**Comment by Luckk56** [comment]
https://x.com/chrettonbase
